### PR TITLE
fix: [2U-391] fixed letters on the Grading scale

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
### **Description**
- fixed formatting of letters inside the grading scale
- copies of translations removed

[YouTrack](https://youtrack.raccoongang.com/issue/2U-391)

### CODE REVIEW

File: src/grading-settings/grading-scale/GradingScale.jsx [hunk 1]
```diff
@@ -144,7 +144,7 @@ const GradingScale = ({
       const updatedLetters = [...prevLetters];
       updatedLetters.splice(updatedLetters.length - 1, 1);
 
-      return updatedLetters.length === 1 ? ['pass'] : updatedLetters;
+      return updatedLetters.length === 1 ? ['Pass'] : updatedLetters;
```
Comment (wowkalucky) [inline src/grading-settings/grading-scale/GradingScale.jsx:147]: Hardcode best practices from legacy...
It'd be nice to extract this label as translatable constant.

File: src/CourseAuthoringRoutes.jsx [hunk 2]
```diff
@@ -99,10 +99,7 @@ const CourseAuthoringRoutes = ({ courseId }) => {
             )}
         </PageRoute>
         <PageRoute path={`${path}/settings/grading`}>
-          {process.env.ENABLE_NEW_GRADING_PAGE === 'true'
-            && (
-              <GradingSettings courseId={courseId} />
-            )}
+          <GradingSettings courseId={courseId} />
```
Comment (ruzniaievdm) [inline src/CourseAuthoringRoutes.jsx:102]: do we need to remove `ENABLE_NEW_GRADING_PAGE` from env either had already been removed?

**Review by ruzniaievdm — APPROVED** [review]

File: src/CourseAuthoringRoutes.jsx (hunk 2 above)
Comment (PKulkoRaccoonGang) [reply src/CourseAuthoringRoutes.jsx]: Removed

File: src/grading-settings/grading-scale/GradingScale.jsx (hunk 1 above)
Comment (PKulkoRaccoonGang) [reply src/grading-settings/grading-scale/GradingScale.jsx]: Yes, your are right 👍

**Review by wowkalucky — APPROVED** [review]
OK..

**Review by ruzniaievdm — APPROVED** [review]

**Pull request merged**